### ISSUE TITLE
Revise the end of 3d style transfer section

### ISSUE THREAD

**Issue by znah** [distillpub/post--differentiable-parameterizations#66]
[ ] I propose dropping this:
> The resulting textures combine elements models of the desired style, while preserving the characteristics of the original texture.
> Take as an example the model created by imposing Van Gogh's starry night as style image.
> The resulting texture contains the repetitive and vigorous brush strokes that characterize Van Gogh's work.
> However, despite the style image contains only cold tones, the resulting fur has a warm orange undertone as it is preserved from the original texture.
> Even more interesting is how the eyes of the bunny are preserved when different styles are transfered.
> For example, when the style is obtained from the Van Gogh's painting, the eyes are transformed in a star-like swirl, while if Kandinsky's work is used, they become abstract patterns that still resemble the original eyes.

because:
- Observations described here can change from run to run
- I tend to explain content color preservation to using rather low layer of GoogleNet for content loss

[ ] Need to say more about weighted gram matrices over iterations, and it's implementation with tf.stop_gradients

[ ] Footnote 13 should either link to shapeways material description, or be removed.
